extensions/blocks/calendly/edit.js

```diff
@@ -0,0 +1,253 @@
+/**
+ * External Dependencies
+ */
+import 'url-polyfill';
+import { isEqual } from 'lodash';
+import queryString from 'query-string';
+
+/**
+ * WordPress dependencies
+ */
+import { BlockControls, BlockIcon, InspectorControls } from '@wordpress/block-editor';
+import {
+	Button,
+	ExternalLink,
+	Notice,
+	PanelBody,
+	Placeholder,
+	ToggleControl,
+	Toolbar,
+} from '@wordpress/components';
+import { useState } from '@wordpress/element';
+import { __, _x } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import './editor.scss';
+import icon from './icon';
+import attributeDetails, { getValidatedAttributes } from './attributes';
+import SubmitButton from '../../shared/submit-button';
+import { getAttributesFromEmbedCode } from './utils';
+import BlockStylesPreviewAndSelector from './BlockStylesPreviewAndSelector';
```

Comment:
Should this be 

```suggestion
import { BlockStylesPreviewAndSelector } from './BlockStylesPreviewAndSelector';
```

Tests appear to be failing because of this right now:
https://travis-ci.org/Automattic/jetpack/jobs/637502239?utm_medium=notification&utm_source=github_status

Reply:
I think its actually a casing issue in the file name, which I have fixed.